### PR TITLE
add bytes.center method with tests

### PR DESCRIPTION
**Pull request by JvMaiia**

### CODE REVIEW

File: tests/datatypes/test_bytes.py [hunk 1]
```diff
@@ -231,6 +231,16 @@ def test_contains(self):
...
         """, exits_early=True)
 
+    def test_center(self):
+        self.assertCodeExecution("""
+            print(b'pybee'.center(5))
+            print(b'pybee'.center(True, b'a'))
+            print(b'pybee'.center(13, b'a'))
+            """)
+        self.assertCodeExecution("""
+            print([].center) #Test TypeError invalid byte array
```
Comment (freakboy3742) [inline tests/datatypes/test_bytes.py]: This is an attribute operation on an array - it has nothing to do with byte arrays, and it isn't invoking a method.

File: tests/datatypes/test_bytes.py [hunk 2]
```diff
@@ -231,6 +231,16 @@ def test_contains(self):
             print(['b', 'e'] in b'pybee') #Test TypeError invalid byte array
         """, exits_early=True)
 
+    def test_center(self):
+        self.assertCodeExecution("""
+            print(b'pybee'.center(5))
+            print(b'pybee'.center(True, b'a'))
+            print(b'pybee'.center(13, b'a'))
```
Comment (freakboy3742) [inline tests/datatypes/test_bytes.py]: There's a lot more cases to test here:
* size < length of string
* error handling for non-bytestring fill arguments
* even and odd sizes for both input strings and final string size
* negative size values
* empty bytestrings

**Comment by JvMaiia** [comment]
I improve the tests and the errors messages, thanks for your help!

**Comment by eliasdorneles** [comment]
Hey @JvMaiia, thanks for contributing!
We're having some issues with the build that isn't running correctly -- we'll have a look into it and get back to you!

**Comment by freakboy3742** [comment]
@JvMaiia Apologies for the delays - the build is now working again, and has revealed a problem with one test on Python 3.4.

**Comment by JvMaiia** [comment]
@freakboy3742 oh ok, i'll fix this

**Comment by freakboy3742** [comment]
Fantastic! Thanks for the work on this! (and apologies again for the CI glitch that slowed things down!)

**Comment by JvMaiia** [comment]
Don't worry, @freakboy3742!

**Pull request merged**